segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/ChunkStorage.java

```diff
@@ -89,6 +89,19 @@ public interface ChunkStorage extends AutoCloseable, StatsReporter {
      */
     boolean supportsConcat();
 
+    /**
+     * Gets a value indicating whether this Storage implementation actually stores data durably and supports reading it back.
+     * Generally almost always storage will support durable and stable data writes and subsequent retrival and therefore should return true.
+     * However, In certain special cases like mock storages used for test/performance, implementation may ignore or throw away data
+     * and in such cases should return false.
+     *
+     * @return True or false.
```

Comment:
Can we say, "True if LTS writes and reads data integrity checksums, false if it doesn't."

Reply:
updated.